ruint-macro/src/lib.rs

```diff
@@ -198,4 +202,9 @@ mod tests {
             assert_eq!(0o10_U8, 8_u64.try_into().unwrap());
         }
     }
+
+    #[test]
+    fn test_overflow_during_parsing() {
+        let _ = uint!(258664426012969093929703085429980814127835149614277183275038967946009968870203535512256352201271898244626862047232_U384);
```

Comment:
Using `uint!` in tests is actually a bit cyclical and problematic. I should change that.